src/core/thread/address_resolver.hpp

```diff
@@ -114,6 +114,17 @@ class AddressResolver : public InstanceLocator
      */
     otError UpdateCacheEntry(const Ip6::Address &aEid, Mac::ShortAddress aRloc16);
 
+    /**
+     * This method removes the cache entry for the EID.
+     *
+     * @param[in]  aEid               A reference to the EID.
+     *
+     * @retval OT_ERROR_NONE          Successfully removed the cache entry.
+     * @retval OT_ERROR_NOT_FOUND     No cache entry with @p aEid.
+     *
+     */
+    otError RemoveCacheEntry(const Ip6::Address &aEid);
```

Comment:
nit: how about renaming to `Remove()` and co-locating with the other `Remove()` methods above?

Reply:
Thanks. Updated.